.gitlab/corona-jobs.yml

```diff
@@ -5,12 +5,12 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 #############################################################################
 
-hip_5.1.0_clang_13_0_0 (build and test on corona):
+hip_5.1.1_clang_13_0_0 (build and test on corona):
```

Comment:
This is probably just aesthetic, but shouldn't that be `hip_5_1_1_clang_13_0_0`?

Reply:
All the jobs need to be checked and possibly renamed for consistency.  I will do that after this is merged.

Reply:
Ah. Ok, that makes sense. In that case, just ignore some of the comments below lol :)